suites/rados/singleton-nomsgr/all/cache-fs-trunc.yaml

```diff
@@ -0,0 +1,25 @@
+roles:
+- [mon.0, mds.a, osd.0, osd.1, osd.2, client.0]
+tasks:
+- install:
+- ceph:
+    log-whitelist:
+    - must scrub before tier agent can activate
+- exec:
+    client.0:
+    - ceph osd pool create data_cache 4
+    - ceph osd tier add data data_cache
+    - ceph osd tier cache-mode data_cache writeback
+    - ceph osd tier set-overlay data data_cache
+    - ceph osd pool set data_cache hit_set_type bloom
+    - ceph osd pool set data_cache hit_set_count 8
+    - ceph osd pool set data_cache hit_set_period 3600
+    - ceph osd pool set data size 1
+    - ceph osd pool set metadata size 1
+- ceph-fuse:
+- exec:
+    client.0:
+      - cd $TESTDIR/mnt.* && dd bs=1M if=/dev/zero of=./foo count=100
+      - cd $TESTDIR/mnt.* && truncate --size 50M ./foo
+      - rados -p data_cache cache-flush-evict-all
+      - 'cd $TESTDIR/mnt.* && [ `stat -c%s ./foo` -eq 52428800 ]'
```

Comment:
..and here, verify that the foo object is still non-zero.  if the bug manifests, we get zeros back instead of the correct data.